docs/pages/faq.rst

```diff
@@ -33,3 +33,21 @@ I am getting ``ModuleNotFoundError: No module named 'uavcan'``. Do I need to ins
 Imports fail with ``AttributeError: module 'uavcan...' has no attribute '...'``. What am I doing wrong?
     Remove the legacy library: ``pip uninstall -y uavcan``.
     Read the :ref:`installation` guide for details.
+
+
+I am experiencing slow SLCAN writes on Windows. What can I do?
+    Increasing the process priority to REALTIME (available if the application has administrator privileges) will help.
+    Without administrator privileges, the HIGH priority set by this code will also help with delays in SLCAN performance.
+    Here's an example::
+
+        if sys.platform.startswith("win"):
+            import ctypes, psutil
+
+            # Reconfigure the system timer to run at a higher resolution. This is desirable for the real-time tests.
+            t = ctypes.c_ulong()
+            ctypes.WinDLL("NTDLL.DLL").NtSetTimerResolution(5000, 1, ctypes.byref(t))
+            p = psutil.Process(os.getpid())
+            p.nice(psutil.REALTIME_PRIORITY_CLASS)
+        elif sys.platform.startswith("linux"):
+            p = psutil.Process(os.getpid())
+            p.nice(20)
```

Comment:
This part sets the minimum priority, not the maximum.

- https://linux.die.net/man/3/nice
- https://en.wikipedia.org/wiki/Nice_(Unix)
- https://psutil.readthedocs.io/en/latest/#psutil.Process.nice

```suggestion
            p.nice(-20)
```